ray-kubernetes/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:latest
+
+# If you wish to default to bash shell instead of sh(Bourne) shell:
+#     RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+
+# disable interactive functions, suppress tzdata prompting for geographic location
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt -qq update \
+    && apt-get -qq install --no-install-recommends apt-utils \
+    && apt-get -qq install git curl nano openssh-server \
+    && apt-get -qq install python-opencv
+
+# Install miniconda3
+RUN curl -LO https://repo.continuum.io/miniconda/Miniconda3-4.5.4-Linux-x86_64.sh
+RUN bash Miniconda3-4.5.4-Linux-x86_64.sh -p /miniconda3 -b
+RUN rm Miniconda3-4.5.4-Linux-x86_64.sh
+ENV PATH=/miniconda3/bin:${PATH}
+RUN conda update -y conda
```

Comment:
Do we need this line and the

```
RUN conda config --add channels conda-forge && \
    conda update pip
```

line?

Does anything go wrong if they're removed?

Reply:
Haven't tried but I don't think anything will go wrong. Probably just a bunch of warnings telling you to upgrade.
But I don't see a harm in keeping it if you're just building it once and putting the image up permanently
Only thing that went wrong when I remove some of the previous `apt install`s was `LZ4`, and it wasn't just because of the removal of `GCC`. I just removed it since it sounds like you want a very minimal docker image.
I also just saw your comment about using helm on #2037 , I will probably look into that in the future.

Reply:
Do you want me to delete those lines? I just thought it might be easier to upgrade and download certain things once when you build the image instead of installing stuff for a bunch of pods individually.

Reply:
I have a slight preference for keeping the example as minimal as possible, but I'm ok with leaving them in for now. Down the road we can see what ends up being the most convenient and/or defer to people who have more experience with Kubernetes.